### PR TITLE
Add codeowners for PYREPL

### PR DESCRIPTION
**Pull request by pablogsal**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -43,6 +43,7 @@ Lib/test/test_patma.py        @brandtbucher
 Lib/test/test_type_*.py       @JelleZijlstra
 Lib/test/test_capi/test_misc.py  @markshannon @gvanrossum
 Tools/c-analyzer/             @ericsnowcurrently
+Lib/_pyrepl/*                 @pablogsal @lysnikolaou @ambv
```
Comment (lysnikolaou) [inline .github/CODEOWNERS]: Maybe add the test folder as well?

Comment (hugovk) [inline .github/CODEOWNERS]: Let's stick it before `Lib/test/...` so it's kind of sorted, can help avoid conflicts.

Comment (pablogsal) [reply .github/CODEOWNERS]: Done

Comment (pablogsal) [reply .github/CODEOWNERS]: Done

**Review by hugovk — APPROVED** [review]

**Review by lysnikolaou — APPROVED** [review]

**Pull request merged**